### PR TITLE
chore: add manifest related metrics

### PR DESCRIPTION
**Pull request by MichaelScofield**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## Refer to a related PR or issue link (optional)

https://github.com/GreptimeTeam/greptimedb-enterprise/issues/39

## What's changed and what's your intention?

The metrics of the time to operate the manifest, labeled with "op". Label values are "update", "open" and "checkpoint", each represent the corresponding manifest operation.

Screenshot of "/metrics" output:

image

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.
- [x]  This PR does not require documentation updates.

### CODE REVIEW

**Comment by MichaelScofield** [comment]
@evenyag PTAL

File: src/mito2/src/manifest/action.rs [hunk 1]
```diff
@@ -22,13 +22,14 @@ use snafu::{OptionExt, ResultExt};
 use store_api::manifest::ManifestVersion;
 use store_api::metadata::RegionMetadataRef;
 use store_api::storage::{RegionId, SequenceNumber};
+use strum::Display;
```
Comment (evenyag) [inline src/mito2/src/manifest/action.rs:25]:
```suggestion
```

File: src/mito2/src/manifest/action.rs [hunk 2]
```diff
@@ -22,13 +22,14 @@ use snafu::{OptionExt, ResultExt};
...
 
 use crate::error::{RegionMetadataNotFoundSnafu, Result, SerdeJsonSnafu, Utf8Snafu};
 use crate::sst::file::{FileId, FileMeta};
 use crate::wal::EntryId;
 
 /// Actions that can be applied to region manifest.
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, Display)]
```
Comment (evenyag) [inline src/mito2/src/manifest/action.rs:32]:
```suggestion
#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
```

**Review by evenyag — APPROVED** [review]
Rest LGTM

File: src/mito2/src/manifest/action.rs (hunk 1 above)
Comment (MichaelScofield) [reply src/mito2/src/manifest/action.rs:25]: I'd like to keep the `Display` for enum, it does no harm but maybe useful in the future like in the logging.

**Review by fengjiachun — APPROVED** [review]
LGTM

File: src/mito2/src/manifest/action.rs (hunk 1 above)
Comment (evenyag) [reply src/mito2/src/manifest/action.rs:25]: Though we can use `Debug`.......

**Pull request merged**